server.go

```diff
@@ -79,6 +79,7 @@ func (users *Users) GetUserByEmail(anEmail string) *User {
 	defer users.UserListMutex.Unlock()
 
 	user, _:= users.UserList[anEmail]
+	// Comment
```

Comment:
Is it fine to ignore the error in here?

Reply:
The map is defined as 'map[string]*User' so if there is no value found default value is returned in this case _nil_ .